packages/app/pages/Api/person.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * API page for displaying the Viewer queries on Person objects.
+ *
+ * @package: REA app
+ * @author: Nicholas Roth <Lou3797>
+ * @since: 2018-2-12
+ */
+
+import * as React from "react";
+import "./api.css"
+import getAllPeople from "../../../ui-bindings/Person/getAllPeople";
+import getPersonById from "../../../ui-bindings/Person/getPersonById";
+import { concatArray } from "./common";
+
+export const Person = (props) => {
+  let person = props.person;
+  return(
+    <div>
+      <div>id: {person.id}</div>
+      <div>name: {person.name}</div>
+      <div>type: {person.type}</div>
+      <div>image: {person.image}</div>
+      <div>note: {person.note}</div>
+      <div>ownedEconomicResources: {concatArray(person.ownedEconomicResources)}</div>
+      <div>agentProcesses: {concatArray(person.agentProcesses)}</div>
+      <div>agentPlans: {concatArray(person.agentPlans)}</div>
+      <div>agentEconomicEvents: {concatArray(person.agentEconomicEvents)}</div>
+      <div>agentCommitments: {concatArray(person.agentCommitments)}</div>
+      <div>agentRelationships: {concatArray(person.agentRelationships)}</div>
+      <div>agentRoles: {concatArray(person.agentRoles)}</div>
+      <div>agentRecipess: {concatArray(person.agentRecipes)}</div>
+      <br/>
+    </div>
+  );
+};
+
+/**
+ * The query field for fetching a Person by ID
+ */
+const PersonField = (props) => {
+  return(
+    <div>
+      <form onSubmit={props.onSubmitAction}>
+        ID: <input type="text" name="value" id="idForm"/>
+        <input type="submit" value="Query"/>
+      </form>
+    </div>
+  );
+};
+
+/**
+ * Maps the array of Person to their individual pieces
+ */
+export const GetAllPeople = getAllPeople( ({ people, loading, error}) => {
+
+  if (loading) {
+    return <p>Loading...</p>
+  } else if (error) {
+    return (<p>Error!</p>) // This can be made more specific
```

Comment:
What's the difference between loading and error where error needs parentheses?

Reply:
There isn't a difference. I was experimenting with how to make errors more specific which led me to needing to add parenthesis to the return statement. I just never ended up removing them it seems when I reverted back to simple error messages.